api/include/opentelemetry/trace/span_id.h

```diff
@@ -66,6 +67,16 @@ class SpanId final
     memcpy(dest.data(), rep_, kSize);
   }
 
+  static SpanId GetRandom()
+  {
+    uint8_t buf[kSize] = {0};
```

Comment:
This functionality is also implemented in the SDK (see [/sdk/src/common/random.cc](https://github.com/open-telemetry/opentelemetry-cpp/blob/master/sdk/src/common/random.cc)). I wonder if it would be worth moving that functionality to the API, or if it's better to duplicate it here.

Reply:
I actually don't know. Might be better to ask Reiley. But I no longer possess the laptop to chat with the team members. Could you do that for me? Thanks

Reply:
Actually I am not able to change this PR's title right now. Do you know anyone who's authorized to change this PR from draft to non-draft and potentially merge this?